src/components/forgotPasswordForm/ForgotPasswordForm.js

```diff
@@ -0,0 +1,16 @@
+import styles from "@/styles/logInModal.module.scss";
+
+const ForgotPasswordForm = ({setToLogin}) => {
+  return (
+    <>
+      <h2>Glömt lösenord?</h2>
+      <form>
+        <input required type="email" placeholder="email"></input>
+        <button className={styles.continueButton} type="submit">SKICKA LÖSENORD</button>
```

Comment:
Inputs don't need a closing tag. Maybe it's personal preference, but for any HTML tag without 'children', this looks cleaner to me:
`<input required type="email" placeholder="email" />`

Reply:
Yeah I agree, will change it.